src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class WriterServiceImplTest {
+    private static WriterServiceImpl writerService;
+
+    @BeforeClass
+    public static void beforeClass() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test
+    public void validParams_Ok() throws IOException {
+        writerService.writeToFile("src/test/resources/output.csv", "type,fruit,quantity"
+                + System.lineSeparator() + "b,banana,20");
+        List<String> expected = List.of("type,fruit,quantity","b,banana,20");
+        List<String> actual = Files.readAllLines(Path.of("src/test/resources/output.csv"));
```

Comment:
```suggestion
    public void validParams_Ok() throws IOException {
        String filePath = "src/test/resources/output.csv";
        writerService.writeToFile(filePath, "type,fruit,quantity"
                + System.lineSeparator() + "b,banana,20");
        List<String> expected = List.of("type,fruit,quantity","b,banana,20");
        List<String> actual = Files.readAllLines(Path.of(filePath));
```

Don't we need to compare the expected and actual results?